tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -22,9 +22,24 @@ bootstrap=Ubuntu
 [Ubuntu]
 source=$REPO_URL
 keyring=ubuntu-keyring
-suite=focal
+suite=$UBUNTU_CODENAME
 packages=$PACKAGES $EXTRA_PKGS
 EOF
+
+	if [ "${CONFIDENTIAL_GUEST}" == "yes" ] && [ "${DEB_ARCH}" == "amd64" ]; then
+		mkdir -p $rootfs_dir/etc/apt/trusted.gpg.d/
+		curl -fsSL https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key |
+			gpg --dearmour -o $rootfs_dir/etc/apt/trusted.gpg.d/intel-sgx-deb.gpg
+		sed -i -e "s/bootstrap=Ubuntu/bootstrap=Ubuntu intel-sgx/" $multistrap_conf
+		cat >> $multistrap_conf << EOF
+
+[intel-sgx]
+source=https://download.01.org/intel-sgx/sgx_repo/ubuntu
+suite=$UBUNTU_CODENAME
+packages=libtdx-attest=1.18\*
```

Comment:
We can upgrade to latest 1.20 version which verified works with CoCo: https://github.com/confidential-containers/guest-components/pull/442

Reply:
Updated!